docs/source/quickstart.rst

```diff
@@ -91,187 +103,193 @@ as follows (this example is also included in ``quickstart/mlflow_tracking.py``):
 
-Log to Databricks Community Edition
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+For more information, including a list of supported model flavors, see :ref:`quickstart_drilldown_log_and_load_model`.
 
-Alternatively, sign up for `Databricks Community Edition <https://databricks.com/try-databricks>`_,
-a free service that includes a hosted tracking server. Note that
-Community Edition is intended for quick experimentation rather than production use cases.
-After signing up, run ``databricks configure`` to create a credentials file for MLflow, specifying
-https://community.cloud.databricks.com as the host.
+Running a model from a specific training run
+============================================
 
-To log to the Community Edition server, set the ``MLFLOW_TRACKING_URI`` environment variable
-to "databricks", or add the following to the start of your program:
+To load and run a model stored in a previous run, you can use the ``mlflow.{library_module_name}.load_model`` function. You'll need the run ID of the run that logged the model. You can find the run ID in the tracking UI:
 
-  .. code-section::
+.. image:: _static/images/quickstart_run_id.png
+    :width: 400px
+    :align: center
+    :alt: Screenshot showing location of run ID in tracking UI
+
+.. code-section::
 
     .. code-block:: python
 
         import mlflow
 
-        mlflow.set_tracking_uri("databricks")
-        # Note: on Databricks, the experiment name passed to set_experiment must be a valid path
-        # in the workspace, like '/Users/<your-username>/my-experiment'. See
-        # https://docs.databricks.com/user-guide/workspace.html for more info.
-        mlflow.set_experiment("/my-experiment")
-
-    .. code-block:: R
-
-        library(mlflow)
-        install_mlflow()
-        mlflow_set_tracking_uri("databricks")
-        # Note: on Databricks, the experiment name passed to mlflow_set_experiment must be a
-        # valid path in the workspace, like '/Users/<your-username>/my-experiment'.  See
-        # https://docs.databricks.com/user-guide/workspace.html for more info.
-        mlflow_set_experiment("/my-experiment")
+        from sklearn.model_selection import train_test_split
+        from sklearn.datasets import load_diabetes
+
+        db = load_diabetes()
+        X_train, X_test, y_train, y_test = train_test_split(db.data, db.target)
+
+        model = mlflow.sklearn.load_model("runs:/97f4e98c5f3645c8acd800cbddf5f6da/model")
+        predictions = model.predict(X_test)
+        print(predictions)
+
+Note that while ``log_model`` saves environment-specifying files such as **conda.yaml** and **requirements.txt**, ``load_model`` does not automatically recreate that environment. To do so, you need to use your preferred method (**conda**, **virtualenv**, **pip**, etc.), using the artifacts saved by ``log_model``.
+
+To learn more about loading runs, see :ref:`quickstart_drilldown_log_and_load_model`.
+
```

Comment:
It might be nice to show the programmatic access pattern for retrieving the model_uri via the run context:

```
with mlflow.start_run() as run:
  mlflow.sklearn.log_model(
    my_model,
    artifact_path="model",
  )


model_uri = f"runs:/{run.info.run_id}/model"
loaded_model = mlflow.sklearn.load_model(model_uri)
```

Reply:
TODO